.travis.yml

```diff
@@ -22,16 +22,8 @@ rvm:
 before_install:
   - gem update --system
   - |
-    r_eng="$(ruby -e 'STDOUT.write RUBY_ENGINE')";
-    if [ "$r_eng" == "jruby" ]; then
-      sudo apt-get update && \
-      sudo apt-get install -y git && \
-      sudo apt-get install -y libpthread-stubs0-dev && \
-      sudo apt-get install -y build-essential && \
-      sudo apt-get install -y zlib1g-dev && \
-      sudo apt-get install -y libssl-dev && \
-      sudo apt-get install -y libsasl2-dev
-    fi
+    sudo apt-get update && \
+    sudo apt-get install -y git build-essential libpthread-stubs0-dev zlib1g-dev libzstd-dev libssl-dev libsasl2-dev
```

Comment:
Won't this break jRuby support?

Reply:
Previously this was enabled only for jRuby. The travis jRuby tests are still passing AFAICT. But I know very little about jRuby or how it might break, so it's worth trying it out.

Reply:
If the tests pass it should be ok, maybe these packages are available by default on bionic.

Reply:
The gem is installed without the need for any dependencies in a MRI docker container with changes in this PR.

```
$ docker run -v $(pwd):/rdkafka -it ruby /bin/bash

root@8b7190045cd6:/# cd rdkafka/ && bundle && cd ext && rake
#=> Success

root@8b7190045cd6:~# uname -a
Linux 8b7190045cd6 4.19.76-linuxkit #1 SMP Tue May 26 11:42:35 UTC 2020 x86_64 GNU/Linux

```

Removing the conditional that installs the dependencies only in JRuby is not necessary but it does not harm either.